content/nextflow-on-aws-batch/40_nextflow202/32_job_queue.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Create Job Queues"
+chapter: false
+weight: 32
+---
+
+## Create Job Queue
+
+Again, let us created a queue via the console.
+![5_queue_workflow_0](/images/nextflow-on-aws-batch/batch/5_queue_workflow_0.png)
+
+To create the job queue we choose a name (**workflow-queue**) a priority of 1 and the map them to the correct compute environment (**od-ce**).
```

Comment:
Change "priority of 1" to bold.

The goal is to let the attendees know in one glance where a keyboard input will be necessary.

Reply:
good catch, done